File: Solutions/0119- Pascal's Triangle II/Solution.py
class Solution:
    def getRow(self, rowIndex: int) -> List[int]:
        my_list = [1]

        if rowIndex == 0:
            return my_list

        while rowIndex > 0:
            temp = my_list
            my_list = [1]
            for i in range(0,len(temp) - 1):
                my_list.append(temp[i] + temp[i+1])
            my_list.append(1)
            rowIndex -= 1

        return my_list

File: Solutions/0119- Pascal's Triangle II/Solution.java
import java.util.ArrayList;
import java.util.List;

class Solution {
    public List<Integer> getRow(int rowIndex) {
        ArrayList<Integer> list = new ArrayList<>();
        list.add(1);

        if (rowIndex == 1)
            return list;

        while (rowIndex > 0) {
            ArrayList<Integer> temp = new ArrayList<>(list);
            list.clear();
            list.add(1);
            for (int i = 0; i < temp.size() - 1; i++) {
                list.add(temp.get(i) + temp.get(i+1));
            }
            list.add(1);
            rowIndex--;
        }
        return list;
    }
}